Update/omot_013.txt

```diff
@@ -2726,7 +2726,7 @@ void main()
 //r…俺にはそれが、嫌な話を聞かせまいとする思いやりからの逡巡だとわかっていたので、逆に安心させるように笑って言った。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…俺にはそれが、嫌な話を聞かせまいとする思いやりからの逡巡だとわかっていたので、逆に安心させるように笑って言った。",
-			NULL, "...Even though I realized that she was trying to avoid telling these bad stories out of concern for me, I smiled and tried to reassure her.", Line_Normal);
+			NULL, "...I realized that she was trying to avoid telling these bad stories out of concern for me, that's why I smiled and tried to reassure her.", Line_Normal);
```

Comment:
```suggestion
			NULL, "...I realized that she was trying to avoid telling these bad stories out of concern for me. That's why I smiled and tried to reassure her.", Line_Normal);
```